docker/parabol-ubi/docker-build/dockerfiles/basic.dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG _NODE_VERSION=${_NODE_VERSION}
+FROM node:${_NODE_VERSION}-bookworm-slim as base
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PORT=3000
+
+# RUN apt update -y && \
```

Comment:
@adaniels-parabol you added systemtap in a [PR](https://github.com/ParabolInc/parabol/pull/9241), but there are no details about why it was added. Could you elaborate a bit more and tell us if we can remove it?

Reply:
It was used for some of the security analysis (inspection scripts) and for some of the security scripts.